packages/shared/src/features/common/components/MobileSocialRegistration.tsx

```diff
@@ -0,0 +1,97 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import type { ButtonProps } from '../../../components/buttons/Button';
+import { Button, ButtonVariant } from '../../../components/buttons/Button';
+import { isIOSNative } from '../../../lib/func';
+import { providerMap, SocialProvider } from '../../../components/auth/common';
+import { useLogContext } from '../../../contexts/LogContext';
+import { AuthEventNames } from '../../../lib/auth';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../../../components/typography/Typography';
+import { cookiePolicy, termsOfService } from '../../../lib/constants';
+import { anchorDefaultRel } from '../../../lib/strings';
+
+interface MobileSocialRegistrationProps {
+  onClick: (provider: SocialProvider) => void;
+  isLoading?: boolean;
+  isDisabled?: boolean;
+}
+
+export function MobileSocialRegistration({
```

Comment:
I think this should be located in onboarding/shared. I wouldn't call it mobile also. just social registration. mobile term is confusing here